src/OpenLoco/Map/TileManager.cpp

```diff
@@ -293,6 +296,156 @@ namespace OpenLoco::Map::TileManager
         }
     }
 
+    // 0x0045F1A7
+    map_pos screenGetMapXY(int16_t x, int16_t y)
+    {
+        registers regs;
+        regs.ax = x;
+        regs.bx = y;
+        call(0x0045F1A7, regs);
+        map_pos pos = { regs.ax, regs.bx };
+        return pos;
+    }
+
+    uint16_t setMapSelectionTiles(int16_t x, int16_t y)
```

Comment:
realise you have copy pasted this but are you able to work out the address of these functions

Reply:
I'll try. I moved them from the construction window. It's my understanding they were part of event functions before.